packages/@aws-cdk/aws-autoscaling/lib/auto-scaling-group.ts

```diff
@@ -703,6 +710,7 @@ export class AutoScalingGroup extends AutoScalingGroupBase implements
 
     this.autoScalingGroup = new CfnAutoScalingGroup(this, 'ASG', asgProps);
     this.osType = imageConfig.osType;
+    this.operatingSystemType = imageConfig.osType;
```

Comment:
Why do we need additional value? can't we use the `osType `? 
Should we add an enum value of `UNDEFINED` to the `OperatingSystemType ` Enum and keep one value?

Reply:
Sure. How about `UNKNOWN` for the third value?

Reply:
sounds good

Reply:
@NetaNir what do you want to do for `OperatingSystemType.UNKNOWN` for this function: https://github.com/aws/aws-cdk/blob/b80c2718055a19a72955e457397d6e812a21e53e/packages/%40aws-cdk/aws-ec2/lib/user-data.ts#L88-L93 ?